android/app/src/main/java/org/fossasia/openevent/OpenEventApp.java

```diff
@@ -106,6 +109,12 @@ public void onCreate() {
         }
         refWatcher = LeakCanary.install(this);
 
+        //Initialize Cache
+        File httpCacheDirectory = new File(getCacheDir(), "picasso-cache");
+        Cache cache = new Cache(httpCacheDirectory, 15 * 1024 * 1024);
+
```

Comment:
@Shailesh351 I think 10 MB would be better, as some individual photos themselves surpass 1 MB